app/src/main/java/at/andreasrohner/spartantimelapserec/preference/preftype/ShowCameraInfoPreference.java

```diff
@@ -152,13 +152,17 @@ public void updateSummary() {
 		} else if ("pref_camera".equals(key)) {
 			String cameraId = prefs.getString("pref_camera", null);
 
-			CameraManager manager = (CameraManager) getContext().getSystemService(Context.CAMERA_SERVICE);
-			try {
-				CameraModel camera = new CameraModel(getContext(), cameraId, manager.getCameraCharacteristics(cameraId));
-				setSummary(camera.toString());
-			} catch (CameraAccessException e) {
-				Log.e(TAG, "Error loading camera details", e);
-				setSummary("Could not load Camera Details: " + e);
+			if (cameraId == null) {
```

Comment:
If the camera is not set yet, a message is show. As this is only on the first start, before anything is configured, it shouldn't be an issue.